firebase-web/src/main/java/io/spine/web/firebase/StoredJson.java

```diff
@@ -100,6 +101,14 @@ public JsonObject asJsonObject() {
         return object.getAsJsonObject();
     }
 
+    /**
+     * Parses this JSON as a message of the given type.
+     */
+    @Internal
+    public <M extends Message> M as(Class<M> cls) {
+        return Json.fromJson(value(), cls);
```

Comment:
Similarly to the one above, let's reference `fromJson` in `@see`.

Also, let's have them both either statically imported, or both used with the class name. Now, it's half-and-half.